### PR TITLE
Actor profile - Replace exporter data in importer page; Use dynamic r…

### PR DESCRIPTION
**Pull request by tiagojsag**
…ounding in volume produced

### CODE REVIEW

File: app/models/actor_attributes.rb [hunk 1]
```diff
@@ -209,8 +226,17 @@ def sustainability
   end
 
   def companies_exporting
+    quant = Quant.find_by(:name => 'SOY_')
+
+    unit = quant.unit
+    value_divisor = 1
+    if (unit.casecmp('tn') == 0)
```
Comment (tiagojsag) [inline app/models/actor_attributes.rb]: This feels like a less hacky hack than simply dividing by 1k all the time.

**Comment by tiagojsag** [comment]
requires https://github.com/Vizzuality/TRASE-frontend/pull/474

File: app/models/actor_attributes.rb [hunk 2]
```diff
@@ -270,13 +296,13 @@ def companies_exporting
...
         id: total['node_id'],
-        y: total['value'].to_f / 1000, # TODO hack
+        y: total['value'].to_f / value_divisor,
         x: indicator_totals_hash[total['node_id']]
       }
     end
 
     {
-      companies_exporting: {
+      companies_sourcing: {
```
Comment (agnessa) [inline app/models/actor_attributes.rb]: not sure why we needed to rename this?

File: app/models/actor_attributes.rb (hunk 1 above)
Comment (agnessa) [reply app/models/actor_attributes.rb]: yes

File: app/models/actor_attributes.rb (hunk 2 above)
Comment (tiagojsag) [reply app/models/actor_attributes.rb]: we dont need, but feels itchy

**Pull request merged**